### PR TITLE
Add ops file to enable bbr

### PR DESCRIPTION
**Pull request by mhuangpivotal**
concourse/concourse#2332

Blocked by: https://github.com/concourse/concourse-bosh-release/pull/29

### CODE REVIEW

File: cluster/operations/bbr.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    sha1: ((bbr_sdk_sha1))
+    version: ((bbr_sdk_version))
+
+- type: replace
+  path: /instance_groups/name=web/jobs/-
+  value:
+    name: bbr-atcdb
+    properties:
+      postgresql:
+        database: atc
```
Comment (cirocosta) [inline cluster/operations/bbr.yml]: Hey,

Do you think we need to make this configurable?

Thanks!

File: cluster/operations/bbr.yml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+        role:
+          name: concourse
```
Comment (cirocosta) [inline cluster/operations/bbr.yml]: Just like in the other comment, this *might* be important to be parametrizable. Wdyt?

thx!

**Comment by cirocosta** [comment]
Hey,

As next steps, @mhuangpivotal and I were looking at some more possible testing that could be done regarding thre three possible ways of having the Database backed up:

- external
- cross-deployment
- intra-deployment

We've been thinking about having the Postgres properties propagated through links to the `bbr-atcdb` job.

thx!

**Comment by taylorsilva** [comment]
Decided to use the bosh links. PR here that updates the bosh release to use bosh links: https://github.com/concourse/concourse-bosh-release/pull/29

This should now allow us to do external, cross-deployment, and intra-deployment testing. These ops files would help us cover intra-deployment testing.

Changed the ops file to use an explicit linking from the web job instead of having any parameters set. Configure once, set everywhere!

File: cluster/operations/backup-atc-colocated-web.yml [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+    sha1: ((bbr_sdk_sha1))
+    version: ((bbr_sdk_version))
+
+- type: replace
+  path: /instance_groups/name=web/jobs/-
+  value:
+    name: bbr-atcdb
+    release: concourse
+    consumes:
+      concourse_db: {from: web}
```
Comment (taylorsilva) [inline cluster/operations/backup-atc-colocated-web.yml]: An implicit link would've worked, so this `consumes` is technically not needed. If we removed them then bosh would just figure out the implicit linking. If we want less yaml then we can delete these lines. Thoughts?

Comment (cirocosta) [reply cluster/operations/backup-atc-colocated-web.yml]: hmmmmm that's interesting - I personally prefer the explicitness that the `from: web` brings - seems like a nice clear way of seeing that the properties are "flowing" from `web` to the `bbr` job.  

Wdyt @YoussB @pivotal-bin-ju ?

Comment (taylorsilva) [reply cluster/operations/backup-atc-colocated-web.yml]: We removed this because you need a matching provides for the consume.

Comment (cirocosta) [reply cluster/operations/backup-atc-colocated-web.yml]: oh, interesting! Thanks!

**Review by cirocosta — APPROVED** [review]
SGTM 👍

**Pull request merged**